mne/viz/ica.py

```diff
@@ -132,7 +133,7 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
                          set_title_and_labels, plot_std, psd_ylabel,
                          spectrum_std, topomap_args, image_args, fig, axes):
     """Plot ICA properties (helper)."""
-    topo_ax, image_ax, erp_ax, spec_ax, var_ax = axes
+    erp_ax, image_ax, spec_ax, topo_ax, var_ax = axes
```

Comment:
That changes also the order of axes passed by the user. An older order is still in the docs of `axes` arg. Ideally the user would not have to change the order (the previous one had a spatial logic to it, which is more intuitive than alphabetical :) ).

Reply:
Hm. Ok. I'll do it.